<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SimplePTTGUI</class>
 <widget class="RollupContents" name="SimplePTTGUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>320</width>
    <height>305</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>320</width>
    <height>155</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>560</width>
    <height>305</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Liberation Sans</family>
    <pointsize>9</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Simple PTT</string>
  </property>
  <widget class="QWidget" name="settingsContainer" native="true">
   <property name="geometry">
    <rect>
     <x>2</x>
     <y>2</y>
     <width>300</width>
     <height>291</height>
    </rect>
   </property>
   <property name="windowTitle">
    <string>Settings</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>3</number>
    </property>
    <property name="leftMargin">
     <number>2</number>
    </property>
    <property name="topMargin">
     <number>2</number>
    </property>
    <property name="rightMargin">
     <number>2</number>
    </property>
    <property name="bottomMargin">
     <number>2</number>
    </property>
    <item>
     <layout class="QHBoxLayout" name="pttLayout">
      <item>
       <widget class="ButtonSwitch" name="startStop">
        <property name="toolTip">
         <string>start/stop acquisition</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../../../sdrgui/resources/res.qrc">
          <normaloff>:/play.png</normaloff>
          <normalon>:/stop.png</normalon>:/play.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="ButtonSwitch" name="ptt">
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>50</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Liberation Sans</family>
          <pointsize>20</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="toolTip">
         <string>Push To Talk</string>
        </property>
        <property name="text">
         <string>PTT</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="statusIndicator">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Idle</string>
        </property>
        <property name="styleSheet">
         <string notr="true">QLabel { background-color: gray; border-radius: 12px; }</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="voxLayout">
      <item>
       <widget class="ButtonSwitch" name="vox">
        <property name="minimumSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Toggle vox system - right click to select audio device</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../../../sdrgui/resources/res.qrc">
          <normaloff>:/audio_mic.png</normaloff>:/audio_mic.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="voxEnable">
        <property name="toolTip">
         <string>Enable vox to control PTT</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDial" name="voxLevel">
        <property name="maximumSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Vox threshold (dB)</string>
        </property>
        <property name="minimum">
         <number>-99</number>
        </property>
        <property name="maximum">
         <number>0</number>
        </property>
        <property name="singleStep">
         <number>1</number>
        </property>
        <property name="pageStep">
         <number>1</number>
        </property>
        <property name="value">
         <number>-20</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="voxLevelText">
        <property name="minimumSize">
         <size>
          <width>30</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>-20</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="audioPeak">
        <property name="minimumSize">
         <size>
          <width>50</width>
          <height>0</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Audio signal peak in dB (with Vox on)</string>
        </property>
        <property name="toolTipDuration">
         <number>14</number>
        </property>
        <property name="text">
         <string>-100 dB</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="voxHold">
        <property name="toolTip">
         <string>Vox hold (ms)</string>
        </property>
        <property name="minimum">
         <number>500</number>
        </property>
        <property name="maximum">
         <number>5000</number>
        </property>
        <property name="singleStep">
         <number>100</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="voxHoldUnits">
        <property name="text">
         <string>ms</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="localDeviceLayout">
      <item>
       <widget class="QPushButton" name="devicesRefresh">
        <property name="maximumSize">
         <size>
          <width>24</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Refresh indexes of available device sets</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../../../sdrgui/resources/res.qrc">
          <normaloff>:/recycle.png</normaloff>:/recycle.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="rxDeviceLabel">
        <property name="text">
         <string>Rx dev</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="rxDevice">
        <property name="minimumSize">
         <size>
          <width>55</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>50</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Receiver deviceset index</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="txDeviceLabel">
        <property name="text">
         <string>Tx dev</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="txDevice">
        <property name="minimumSize">
         <size>
          <width>55</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>50</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Transmitter deviceset index</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="lastCommandLog">
        <property name="maximumSize">
         <size>
          <width>24</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Show last command status and log</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../../../sdrgui/resources/res.qrc">
          <normaloff>:/listing.png</normaloff>:/listing.png</iconset>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="DelaysLayout">
      <item>
       <widget class="QLabel" name="rxtxDelayLabel">
        <property name="text">
         <string>Rx-Tx </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="rxtxDelay">
        <property name="toolTip">
         <string>Rx to Tx transition delay (ms)</string>
        </property>
        <property name="minimum">
         <number>100</number>
        </property>
        <property name="maximum">
         <number>5000</number>
        </property>
        <property name="singleStep">
         <number>100</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="rxtxDelayUnits">
        <property name="text">
         <string>ms</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="txrxDelayLabel">
        <property name="toolTip">
         <string/>
        </property>
        <property name="text">
         <string>Tx-Rx</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="txrxDelay">
        <property name="toolTip">
         <string>Tx to Rx transition delay (ms)</string>
        </property>
        <property name="minimum">
         <number>100</number>
        </property>
        <property name="maximum">
         <number>5000</number>
        </property>
        <property name="singleStep">
         <number>100</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="txrxDelayUnits">
        <property name="text">
         <string>ms</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="commandRxTxLayout">
      <item>
       <widget class="ButtonSwitch" name="commandRxTxEnable">
        <property name="toolTip">
         <string>Enable/Disable Rx to Tx command</string>
        </property>
        <property name="text">
         <string>Rx-Tx Cmd</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="commandRxTxFileDialog">
        <property name="minimumSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Choose Rx to Tx command</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../../../sdrgui/resources/res.qrc">
          <normaloff>:/load.png</normaloff>:/load.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="commandRxTx">
        <property name="toolTip">
         <string>Rx to Tx command</string>
        </property>
        <property name="text">
         <string>...</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="commandTxRxLayout">
      <item>
       <widget class="ButtonSwitch" name="commandTxRxEnable">
        <property name="toolTip">
         <string>Enable/Disable Tx to Rx command</string>
        </property>
        <property name="text">
         <string>Tx-Rx Cmd</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="commandTxRxFileDialog">
        <property name="minimumSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Choose Tx to Rx command</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../../../sdrgui/resources/res.qrc">
          <normaloff>:/load.png</normaloff>:/load.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="commandTxRx">
        <property name="toolTip">
         <string>Tx to Rx command</string>
        </property>
        <property name="text">
         <string>...</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="gpioRxTxLayout">
      <item>
       <widget class="ButtonSwitch" name="gpioRxTxControlEnable">
        <property name="toolTip">
         <string>Enable/Disable Rx to Tx GPIO</string>
        </property>
        <property name="text">
         <string>Rx-Tx GPIO</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="gpioRxTxMaskLabel">
        <property name="toolTip">
         <string>Rx to Tx GPIO mask (1 byte hex)</string>
        </property>
        <property name="text">
         <string>Mask</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="gpioRxTxMask">
        <property name="maximumSize">
         <size>
          <width>35</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Rx to Tx GPIO mask (1 byte hex)</string>
        </property>
        <property name="inputMask">
         <string>HH</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="gpioRxTxValueLabel">
        <property name="toolTip">
         <string>Rx to Tx GPIO value (1 byte hex)</string>
        </property>
        <property name="text">
         <string>Val</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="gpioRxTxValue">
        <property name="maximumSize">
         <size>
          <width>35</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Rx to Tx GPIO value (1 byte hex)</string>
        </property>
        <property name="inputMask">
         <string>HH</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_4">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QRadioButton" name="gpioRxControl">
        <property name="toolTip">
         <string>Rx has GPIO controls</string>
        </property>
        <property name="text">
         <string>Rx</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="gpioTxRxLayout">
      <item>
       <widget class="ButtonSwitch" name="gpioTxRxControlEnable">
        <property name="toolTip">
         <string>Enable/Disable Tx to Rx GPIO</string>
        </property>
        <property name="text">
         <string>Tx-Rx GPIO</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="gpioTxRxMaskLabel">
        <property name="toolTip">
         <string>Tx to Rx GPIO mask (1 byte hex)</string>
        </property>
        <property name="text">
         <string>Mask</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="gpioTxRxMask">
        <property name="maximumSize">
         <size>
          <width>35</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Tx to Rx GPIO mask (1 byte hex)</string>
        </property>
        <property name="inputMask">
         <string>HH</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="gpioTxRxValueLabel">
        <property name="toolTip">
         <string>Tx to Rx GPIO value (1 byte hex)</string>
        </property>
        <property name="text">
         <string>Val</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="gpioTxRxValue">
        <property name="maximumSize">
         <size>
          <width>35</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Tx to Rx GPIO value (1 byte hex)</string>
        </property>
        <property name="inputMask">
         <string>HH</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_5">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QRadioButton" name="gpioTxControl">
        <property name="toolTip">
         <string>Tx has GPIO controls</string>
        </property>
        <property name="text">
         <string>Tx</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ButtonSwitch</class>
   <extends>QToolButton</extends>
   <header>gui/buttonswitch.h</header>
  </customwidget>
  <customwidget>
   <class>RollupContents</class>
   <extends>QWidget</extends>
   <header>gui/rollupcontents.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../../sdrgui/resources/res.qrc"/>
 </resources>
 <connections/>
</ui>
